### ISSUE TITLE
Sub Collection Facet missing?

### ISSUE THREAD

**Issue by jgreidy** [cul-it/singlecore-library-cornell-edu#1030]
sub_coll_tesim is missing from the MAP spreadsheet. It had been used as a hidden facet, and in the item record.
Example records: 550786, 549509
https://digital.library.cornell.edu/catalog/ss:550786

This is to confirm that this field is not needed.

**Comment by mhk33** [comment]
It looks like have sub_coll_tesim in 3 collections:

1. Vicos - it seems the field was removed/is not used any more and is not needed
2. Indonesian Music Archive (Gamelan) - field was repurposed as "Series". I will update the ini so this displays correctly. Therefore we don't need Subcollection in Gamelan.
3. RMC Images - Subcollection is still in use and has values. I don't know what field this should be mapped to in the MAP. @cm647 do you have any thoughts on this?

**Comment by cm647** [comment]
I think they're conceptually similar. They ones for Adler & Bam say "Cornell University Hip Hop Collection", the antiquities ones say "Cornell Collection of Antiquities" and here are the values in the RMC glob of content. They're less neat/precise, but do give a similar sense of intermediary collection level between RMC & box-level MD. What do you think? :

Human Sexuality
Cornell Graphic records 1923-1926
Cornell University faculty biographical   files 1865-2004
Aggregated Cornell photographs, ca.   1868-1986
Lucien L. Nunn papers 1863-1971
Harper's weekly
University Archives
Early Chinese export watercolors
Wason Collection on East Asia
Public Affairs Records
Photo Sciences Negatives
Huntington Free Library
Asia Rare
Archives
Rare Books
Manuscripts
History of Science
Icelandic
Petrarch
Adelmann
Fiske Icelandic Collection
Fiske Dante Collection
Witchcraft
Sparks manuscripts
Wordsworth
Hill
Uris Library
Library Annex
French Revolution
Slavery
Rare Books\|History of Science

**Comment by mhk33** [comment]
Some of those values make me think of the Archival Collection, too. I guess I am a bit hesitant to decide because it's RMC and they seem to have specific uses for different fields. But I don't know how much Jasmine would just make the call, or consult with RMC. And the RMC Images metadata has not been very structured in the past.

**Comment by cm647** [comment]
I don’t know either! Is this something that needs to be decided now and I should kick up the chain to someone RMCish? Or can it lie fallow until Jasmine gets back? I agree that there’s sort of a scale problem between collecting program and archival collection.


Chloe A. McLaren
Metadata Projects Librarian
Library Technical Services
Cornell University Library

**Comment by mhk33** [comment]
Jim - maybe we just leave sub_coll_tesim out for now, and we circle back with Jasmine on what to do with Subcollection for RMC, when she's back.

**Comment by jgreidy** [comment]
I'll make Subcollection visible in the item view for now. 
sub_coll_tesim has a facet that has :show => false. I assume that is to make it possible to have :link_to_search => true in the add_show_field, in other words, the facet does not show up, but you can click on the value in the item view to do a search for other things in the same subcollection.

**Comment by jgreidy** [comment]
Actually, since sub_coll_tesim is not in the MAP spreadsheet, it does not get included in the mapping. (The mapping is generated automatically from the spreadsheet.) I put the display code in singlecore in case it gets added to the spreadsheet.

**Comment by mhk33** [comment]
> sub_coll_tesim has a facet that has :show => false. I assume that is to make it possible to have :link_to_search => true in the add_show_field, in other words, the facet does not show up, but you can click on the value in the item view to do a search for other things in the same subcollection.

Yes, @jgreidy, that is right. You may see :show => on a few other fields as well; same reasoning.
